local/kcp/start.sh

```diff
@@ -42,8 +42,8 @@ kcp-binaries() {
     git clone https://github.com/kcp-dev/kcp.git
     KCP_DIR="${KCP_PARENT_DIR}/kcp"
     pushd kcp
-    KCP_BRANCH="${KCP_BRANCH:-release-0.7}"
-    git checkout "${KCP_BRANCH}"
+    KCP_BRANCH="${KCP_BRANCH:-v0.8.0}"
+    git checkout tags/"${KCP_BRANCH}" -b "${KCP_BRANCH}"
```

Comment:
We should probably rename KCP_BRANCH to KCP_TAG. That could be done in another PR.